scripts/west_commands/sdk.py

```diff
@@ -0,0 +1,518 @@
+
+                --personal-access-token specifies the GitHub personal access token.
+                This helps to relax the limits on the number of REST API calls.
+
+                --api-url specifies the REST API endpoint for GitHub releases information
+                when installing the SDK from a different GitHub repository."""
+            ),
+        )
+
+        parser.add_argument(
+            "subcommand",
+            nargs="?",
+            default="list",
+            choices=["list", "install"],
+            help="select a subcommand. 'list' is the default.",
+        )
+        parser.add_argument(
+            "-v",
+            "--version",
+            default=None,
+            nargs="?",
+            metavar="SDK_VER",
+            help="specifies the install version of Zephyr SDK."
+            "If not specified, the install version detects from "
+            "${ZEPHYR_BASE}/SDK_VERSION file.",
+        )
+        parser.add_argument(
+            "-d",
+            "--install-dir",
+            default=None,
+            metavar="DIR",
+            help="specifies the install destination directory.",
+        )
+        parser.add_argument(
+            "-i",
+            "--interactive",
+            action="store_true",
+            help="launches installer in interactive mode. "
+            "--toolchains, --no-toolchains and --no-hosttools will be ignored if this option enabled.",
+        )
+        parser.add_argument(
+            "-t",
+            "--toolchains",
+            metavar="TC",
+            nargs="*",
+            help="specifies the installation of toolchains. "
+            "If neither is specified, all toolchains will be installed.",
+        )
+        parser.add_argument(
+            "-T",
+            "--no-toolchains",
+            action="store_true",
+            help="do not install toolchains. "
+            "--toolchains will be ignored if this option is enabled.",
+        )
+        parser.add_argument(
+            "-H",
+            "--no-hosttools",
+            action="store_true",
+            help="do not install host-tools.",
+        )
+        parser.add_argument(
+            "--personal-access-token", help="specify GitHub personal access token."
+        )
+        parser.add_argument(
+            "--api-url",
+            default="https://api.github.com/repos/zephyrproject-rtos/sdk-ng/releases",
+            help="specifies the GitHub releases API endpoint for finding SDKs.",
+        )
+
+        return parser
+
+    def os_arch_name(self):
+        system = platform.system()
+        machine = platform.machine()
+
+        if system == "Linux":
+            osname = "linux"
+        elif system == "Darwin":
+            osname = "macos"
+        elif system == "Windows":
+            osname = "windows"
+        else:
+            self.die(f"Unsupported system: {system}")
+
+        if machine == "aarch64":
```

Comment:
Works on macOS if I add `arm64` (Apple silicon)

```suggestion
        if machine in ["aarch64", "arm64"]:
```

Reply:
Thank you for verifying, and I am sorry for taking your time.
Fixed it.